packages/cli/test/services/project-initializer.test.ts

```diff
@@ -35,9 +35,6 @@ describe('project initializer', (): void => {
     repository: '',
     providerPackageName: defaultProvider,
     boosterVersion: '0.5.1',
-    default: true,
-    skipInstall: false,
-    skipGit: false,
```

Comment:
It looks like these changes were left from previous commits.

```suggestion
    skipInstall: false,
    skipGit: false,
    interactive: false,
```

Reply:
It could make sense to properly tag the assigned variable type to make the compiler detect type mismatches.

```typescript
const defaultProjectInitializerConfig: ProjectInitializerConfig = {
  ...
}
```

Instead of casting the type with the `as` at the end of the assignment.